### PR TITLE
BOM-1519

### PR DESCRIPTION
**Pull request by awais786**
https://openedx.atlassian.net/browse/BOM-1519
Removing django<2.2 versions from tox/travis.
Adding BOM-1533 python3.8 requirement also.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,9 +35,11 @@ matrix:
         - docker exec -t discovery bash -c 'source /edx/app/discovery/discovery_env && cd /edx/app/discovery/discovery/ && make static'
         - docker exec -t discovery bash -c 'source /edx/app/discovery/discovery_env && cd /edx/app/discovery/discovery/ && make quality'
         - docker exec -t discovery bash -c 'source /edx/app/discovery/discovery_env && cd /edx/app/discovery/discovery/ && make check_keywords'
-    - env: TOXENV=django20
-    - env: TOXENV=django21
-    - env: TOXENV=django22
+    - python: 3.5
+      env: TOXENV=django22
```
Comment (mikix) [inline .travis.yml]: This is a duplicate, right? You have same config above.

File: .travis.yml [hunk 2]
```diff
@@ -21,7 +21,9 @@ before_install:
 
 matrix:
   include:
-    - env: TOXENV=django111
+    - python: 3.5
```
Comment (mikix) [inline .travis.yml]: You don't need to specify python here, there's a global python version defined, right?

File: .travis.yml (hunk 1 above)
Comment (awais786) [reply .travis.yml]: Yes, thanks for catching this.

File: .travis.yml (hunk 2 above)
Comment (awais786) [reply .travis.yml]: I added just for readability. I have removed it.

**Review by mikix — APPROVED** [review]

**Pull request merged**